frontend/util.lua

```diff
@@ -723,8 +723,15 @@ end
 --- Basically a helper method to check a specific list of file extensions.
 ---- @string filename
 ---- @treturn boolean
-function util.isAllowedScript(file)
+function util.isAllowedScript(file, has_restricted_scripts)
```

Comment:
remove the entire function. Move comment to generic `Device:canExecuteScript(file)`